src/renderer/components/publication/Cover.tsx

```diff
@@ -27,15 +27,6 @@ export default class Cover extends React.Component<ICoverProps, null> {
 
         if (this.props.publication.cover == null) {
             let authors = "";
-            const bodyCSS = Object.assign({}, Styles.BookCover.body);
-            let colors = this.props.publication.customCover;
```

Comment:
If I am not mistaken, in this PR `publication.customCover` becomes an unused feature.
I suggest to:
1) Either - reinstate support for `publication.customCover`.
2) Or - eliminate dead code, such as (but not limited to):
https://github.com/readium/readium-desktop/blob/cd3997f902be46d4299dc97e0f3ccbdd09fbe69c/src/common/models/custom-cover.ts#L8-L26
https://github.com/readium/readium-desktop/blob/cd3997f902be46d4299dc97e0f3ccbdd09fbe69c/src/main/services/catalog.ts#L332-L339

The latter option has repercussions on the database schema. I do not know the technical details well-enough to assess whether or not this would introduce risks of regression bugs (i.e. handling existing database storage with different schema / parsing rules).